homing_missiles.py

```diff
@@ -1,40 +1,64 @@
 import pygame
-from wordgenerator import WordGenerator
+
 from targeting_terminal import TargetingTerminal
+from wordgenerator import WordGenerator
 
 pygame.font.init()
 
+"""
+Colours
+"""
 BLACK = 0, 0, 0
-RED = 255, 0, 0
-GREEN = 0, 255, 0
 BLUE = 0, 0, 255
-YELLOW = 255, 255, 0
+GREEN = 0, 255, 0
+RED = 255, 0, 0
 WHITE = 255, 255, 255
+YELLOW = 255, 255, 0
+
+"""
+Event Keys
+"""
 EVENT_KEY_a = 97
-EVENT_KEY_z = 122
-EVENT_KEY_ENTER = 13
 EVENT_KEY_BACKSPACE = 8
+EVENT_KEY_ENTER = 13
+EVENT_KEY_z = 122
 
 
 class HomingMissilesTargetingSystem():
 
     def __init__(self, universe, DRAWING_SCALE):
-        self.word_generator_ = WordGenerator()
         self.universe_ = universe
         self.DRAWING_SCALE_ = DRAWING_SCALE
-        self.ui_font_ = pygame.font.SysFont("monospace", 15*DRAWING_SCALE)
+
+        self.current_text_ = ""
         self.enemy_color_ = YELLOW
+        self.font_size_ = 15
         self.main_character_color_ = GREEN
         self.target_tags_ = dict()
-        self.current_text_ = ""
+        self.target_tag_y_spacing_ = 5
         self.targeting_terminal_ = TargetingTerminal(DRAWING_SCALE)
+        self.text_antialias_ = 1
```

Comment:
https://www.pygame.org/docs/ref/font.html#pygame.font.Font.render 
From the docs, the second argument is antialias, to be honest I don't know much about what the "1" does though.